PopPUNK/models.py

```diff
@@ -724,6 +725,15 @@ def fit(self, X, sample_names, model, max_move, min_move, startFile = None, indi
         self.min_move = min_move
         self.unconstrained = unconstrained
 
+        # load CUDA libraries
+        if use_gpu:
+            try:
+                import cugraph
+                import cudf
+            except ImportError as e:
+                sys.stderr.write("cugraph and cudf unavailable\n")
+                raise ImportError(e)
```

Comment:
I would prefer to keep the imports at global scope if possible (if this might be called multiple times). You could import at the top the same way as here, but set a variable to False if failed, and exit here

Reply:
Done in e18ed35.